### ISSUE TITLE
Feature request: Revoking User ID

### ISSUE THREAD

**Issue by Yonle** [oxen-io/session-android#953]
In case if our account is attacked by another anonymous users, We could revoke our uid to solve the issue but not by recreating a new account.

**Comment by hjubb** [comment]
When the open group ID blinding is switched on from the server-side the issue of session ID discoverability and anti-spam should be mitigated a lot better. Are you referring to creating a new Session ID without having to clear the data in the app?

**Comment by Yonle** [comment]
> Are you referring to creating a new Session ID without having to clear the data in the app?

Yes. It should also inform to selected contact that "my" uid has been changed

**Comment by KeeJef** [comment]
Could be useful for rotation of Session ID's for security purposes too
